bin/run_docker.py

```diff
@@ -86,23 +149,34 @@ def handle_outputs(output_path: str, output_file_name: str, log_text: str):
     # Glob any expected output files stored in the output_dir as a result of the container run
     file_glob = glob(os.path.join(output_path, output_file_name + ".*"))
 
+    bad_output_msg = None
+
+    # First check if an incorrect number of output files were generated
     if len(file_glob) != 1:
-        bad_output_msg = f"Expected 1 Docker container output file with base name '{output_file_name}' in the output directory. Got {len(file_glob)}, or a file incorrectly named. If multiple output files are generated, please zip them into a single file for processing."
+        bad_output_msg = f"Expected 1 Docker container output file with base name '{output_file_name}' in the output directory. Got {len(file_glob)}, or a file incorrectly named. If multiple output files are generated, please zip them into a single file for processing"
```

Comment:
Did you mean for this period to be removed?

Reply:
Yep, I realized the periods in these messages looked weird because I had a sentence period in the wrapper that these strings get called in, so it looks like `"blah.'."` so I removed them all in this PR.